docs/pages/enterprise/workflow/ssh-approval-email.mdx

```diff
@@ -0,0 +1,327 @@
+---
+title: Teleport integration with Email
+description: This guide explains how to setup a Email plugin for Teleport for privilege elevation approval notification.
+h1: Teleport Email Plugin Setup
+---
+
+This guide will talk through how to setup Teleport with Email notification for Access Requests.
+
+
+## Setup
```

Comment:
We should use the H2 format that other guides use: a `## Prerequisites` section followed by H2 headings in the format `## Step n/d. Do something`. This way, readers can tell how far they are into the guide.